capstone/src/main/java/com/google/sps/servlets/DashboardHandlerServlet.java

```diff
@@ -27,31 +31,28 @@
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-        
-        // TODO: In a later CL we will make it so that it only gets User-specific classrooms
-        PreparedQuery results = datastore.prepare(new Query(Classroom.CLASSROOM_ENTITY_NAME));
+        Key userKey = ((User) request.getAttribute(User.USER_ENTITY_NAME)).getUserKey();
+        Filter roomFilter = new FilterPredicate(Room.HOST_PROPERTY_KEY, FilterOperator.EQUAL, userKey);
+        Query query = new Query(Room.ROOM_ENTITY_NAME).setFilter(roomFilter);
+        PreparedQuery results = datastore.prepare(query);
```

Comment:
This change should probably be part of another PR. The logical group of changes this PR has (changing the naming/keys/etc for Classroom) is unrelated to this change, so they should be separate PRs. This could even be just a super duper simple 4 line PR which adds this feature

Reply:
I see. Originally, I was going to make this as a 13 line PR but I decided against it since it was so small. Now in the future, I'll keep note to add PRs like this.